src/parser/Parser.spec.ts

```diff
@@ -14,6 +14,31 @@ describe('parser', () => {
         });
     });
 
+    describe('findReferences', () => {
+        it('gets called if references are missing', () => {
```

Comment:
Here's a great example of how a plugin would request the parser to regenerate the references.